code/game/antagonist/antagonist_print.dm

```diff
@@ -10,6 +10,9 @@
 		if(P.ambitions)
 			text += "<br><font color='purple'><b>Their goals for today were:</b></font>"
 			text += "<br>  '[P.ambitions]'"
+		if(P.current.stat == DEAD && P.last_words)
+			text += "<br><b>Their last words were:</b></font>"
+			text += "<br>  '[P.last_words]'"
```

Comment:
Perhaps don't make it two separate lines? There's already a shitton of stuff displayed at round end (along with a ton of chat spam), I think keeping it shorter would be better.